lib/shower/default-options.js

```diff
@@ -1,17 +1,13 @@
 export default {
     container_selector: '.shower',
-
-    debug_mode: false,
-    debug_mode_classname: 'debug',
-
-    sessionstore_key: 'shower',
-
-    slides_selector: '.shower .slide',
+    slides_selector: '.slide',
 
     mode_full_classname: 'full',
     mode_list_classname: 'list',
 
-    slide_title_element_selector: 'h2',
-    slide_active_classname: 'active',
-    slide_visited_classname: 'visited',
+    slide: {
+        titleElementSelector: 'h2',
+        activeClass: 'active',
```

Comment:
I've renamed `classname` => `class` ("name" part is not super useful) and camelCased (for better linting & destructuring).